### PR TITLE
Improve k8s documentation as it become obsolete with the latest changes to the Ansible scripts bug

### PR DESCRIPTION
**Pull request by jacobdotcosta**
Improved documentation due to changes to the Ansible Inventory format. Fixed tag inheritance problems in import_role tasks.

### CODE REVIEW

File: ansible/playbook/README.md [hunk 1]
```diff
@@ -108,7 +124,7 @@ The section describes how to maintain hosts in Ansible and their use. It's execu
 
 ## Create a host
 
-To create a host first the host must be added to the Ansible inventory. This is done using the following statement:
```
Comment (cmoulliard) [inline ansible/playbook/README.md]: I think that we should be here more didactic and explain what will happen when we execute the playbook mentioned for our project. If I remember correctly, the file `ansible/inventory/hosts.yml` needs to be created first - is it correct ? Is it documented like also how the structure should be `all: children : ...` ? how we could add additional var for an host. Dont forget to add real example please

File: ansible/playbook/README.md [hunk 2]
```diff
@@ -45,12 +52,27 @@ Ansible works against multiple systems in an infrastructure, called “hosts”,
...
+* `hosts.yml`: YAML file with group structure information as well as group variable assignment
+* `pass_inventory.py`: Python script that dynamically builds the inventory from the `passwordstore` project for use in the Ansible Playbooks. This task is done automatically when Ansible is executed. 
 
-The inventory is built from the `passwordstore` project using the `pass_inventory.py` python script. This task is done automatically when Ansible is executed. 
 For more information on the `passwordstore` project check gitlab.
 
 More information on the Ansible Inventory and how to build it in the [Ansible User Guide](https://docs.ansible.com/ansible/latest/user_guide/intro_inventory.html).
 
+# `hosts.yml`
+
```
Comment (cmoulliard) [inline ansible/playbook/README.md]: Can you add an example about the `hosts.yml` to be created as most of us have no idea about what it should contain ? @jacobdotcosta

File: ansible/playbook/README.md [hunk 3]
```diff
@@ -106,14 +108,6 @@ More information on these documents:
 
 This system already includes a static inventory.
```
Comment (cmoulliard) [inline ansible/playbook/README.md]: Can you add the link to the resource on github please ? @jacobdotcosta

Comment (jacobdotcosta) [reply ansible/playbook/README.md]: done.

**Comment by cmoulliard** [comment]
LGTM. Can be merged. I will read all the documentation after you will merge it

**Pull request merged**